torch/onnx/symbolic_opset9.py

```diff
@@ -1392,7 +1392,10 @@ def detach(g, input):
     return input
 
 
-def contiguous(g, input):
+@parse_args('v', 'i')
+def contiguous(g, input, memory_format):
+    if memory_format != 2:  # default value torch.contiguous_format
```

Comment:
Hmm, does it have to be an integer comparison here? (I don't actually know, just wondering)

Reply:
Unfortunately is it jit traced int constant at this point already. That is why I additionally commented it.